config/config.h.cmake

```diff
@@ -1,329 +1,349 @@
 /* config/config.h.cmake.  Hand crafted from config/config.h.in.  */
-/* config/config.h.in.  Generated from configure.ac by autoheader.  */
 
 /* Define if building universal (internal helper macro) */
-#cmakedefine AC_APPLE_UNIVERSAL_BUILD
```

Comment:
What does adding 1 here do? I guess it should be #cmakedefine01 if we need 1/0?

Reply:
This results in `#define VARIABLE 1` being written for true values and `/* #undef VARIABLE */` otherwise. 

`#cmakedefine01`  adds `#define VARIABLE 0` for false values instead, which breaks `#if #defined(VARIABLE)` and `#ifdef VARIABLE` checks, which are quite common.

Reply:
Oh, and as for adding the "1" instead of leaving it as a plain `#define` - some locations in the code use `#if VARIABLE` instead of `#if defined(VARIABLE)` for checks. Also, it matches the behavior of `configure.ac` and is a bit safer as a result.

Reply:
Got it, thanks!